src/diffusers/models/autoencoders/autoencoder_kl.py

```diff
@@ -426,14 +434,14 @@ def tiled_decode(self, z: torch.Tensor, return_dict: bool = True) -> Union[Decod
 
     def forward(
         self,
-        sample: torch.Tensor,
+        sample: torch.FloatTensor,
```

Comment:
```suggestion
        sample: torch.Tensor,
```

Reply:
I think we had decided to use `torch.FloatTensor` going forward